README.md

```diff
@@ -1,4 +1,10 @@
 # nodejs-collection
 A place for the Node.js Collective to organize the editing process
 
+## Tone and Content
+
+We are looking for posts that inform, teach, experiment, and generally enrich our community. There's no hard and fast rule of what an article should or feel like; Node.js Collection articles should be as diverse as the community itself. Ideally, contributions should encourage readers to explore new ideas, dig deeper into old ones, and to inspire readers to experiment and tinker. Articles can cover any number of topics such as:
```

Comment:
typo: `an article should or feel like` 

(Also: .md files in our repos tend to get wrapped at 80 chars, but I'm certainly OK if this repo doesn't want to do that. Just pointing it out if you want to conform for the sake of uniformity.)